ui/litellm-dashboard/src/components/provider_info_helpers.tsx

```diff
@@ -2,171 +2,172 @@ import OpenAI from "openai";
-    Azure = "Azure",
-    Azure_AI_Studio = "Azure AI Foundry (Studio)",
-    Anthropic = "Anthropic",
-    Vertex_AI = "Vertex AI (Anthropic, Gemini, etc.)",
-    Google_AI_Studio = "Google AI Studio",
-    Bedrock = "Amazon Bedrock",
-    Groq = "Groq",
-    MistralAI = "Mistral AI",
-    Deepseek = "Deepseek",
-    Cohere = "Cohere",
-    Databricks = "Databricks",
-    Ollama = "Ollama",
-    xAI = "xAI",
-    AssemblyAI = "AssemblyAI",
-    Cerebras = "Cerebras",
-    Sambanova = "Sambanova",
-    Perplexity = "Perplexity",
-    TogetherAI = "TogetherAI",
-    Openrouter = "Openrouter",
-    FireworksAI = "Fireworks AI"
+  OpenAI = "OpenAI",
+  OpenAI_Compatible = "OpenAI-Compatible Endpoints (Together AI, etc.)",
+  OpenAI_Text = "OpenAI Text Completion",
+  OpenAI_Text_Compatible = "OpenAI-Compatible Text Completion Models (Together AI, etc.)",
+  Azure = "Azure",
+  Azure_AI_Studio = "Azure AI Foundry (Studio)",
+  Anthropic = "Anthropic",
+  Vertex_AI = "Vertex AI (Anthropic, Gemini, etc.)",
+  Google_AI_Studio = "Google AI Studio",
+  Bedrock = "Amazon Bedrock",
+  Groq = "Groq",
+  MistralAI = "Mistral AI",
+  Deepseek = "Deepseek",
+  Cohere = "Cohere",
+  Databricks = "Databricks",
+  Ollama = "Ollama",
+  xAI = "xAI",
+  AssemblyAI = "AssemblyAI",
+  Cerebras = "Cerebras",
+  Sambanova = "Sambanova",
+  Perplexity = "Perplexity",
+  TogetherAI = "TogetherAI",
+  Openrouter = "Openrouter",
+  FireworksAI = "Fireworks AI"
+
+}
 
-  }
-  
 export const provider_map: Record<string, string> = {
-    OpenAI: "openai",
-    OpenAI_Text: "text-completion-openai",
-    Azure: "azure",
-    Azure_AI_Studio: "azure_ai",
-    Anthropic: "anthropic",
-    Google_AI_Studio: "gemini",
-    Bedrock: "bedrock",
-    Groq: "groq",
-    MistralAI: "mistral",
-    Cohere: "cohere_chat",
-    OpenAI_Compatible: "openai",
-    OpenAI_Text_Compatible: "text-completion-openai",
-    Vertex_AI: "vertex_ai",
-    Databricks: "databricks",
-    xAI: "xai",
-    Deepseek: "deepseek",
-    Ollama: "ollama",
-    AssemblyAI: "assemblyai",
-    Cerebras: "cerebras",
-    Sambanova: "sambanova",
-    Perplexity: "perplexity",
-    TogetherAI: "togetherai",
-    Openrouter: "openrouter",
-    FireworksAI: "fireworks_ai"
+  OpenAI: "openai",
+  OpenAI_Text: "text-completion-openai",
+  Azure: "azure",
+  Azure_AI_Studio: "azure_ai",
+  Anthropic: "anthropic",
+  Google_AI_Studio: "gemini",
+  Bedrock: "bedrock",
+  Groq: "groq",
+  MistralAI: "mistral",
+  Cohere: "cohere_chat",
+  OpenAI_Compatible: "openai",
+  OpenAI_Text_Compatible: "text-completion-openai",
+  Vertex_AI: "vertex_ai",
+  Databricks: "databricks",
+  xAI: "xai",
+  Deepseek: "deepseek",
+  Ollama: "ollama",
+  AssemblyAI: "assemblyai",
+  Cerebras: "cerebras",
+  Sambanova: "sambanova",
+  Perplexity: "perplexity",
+  TogetherAI: "togetherai",
+  Openrouter: "openrouter",
+  FireworksAI: "fireworks_ai"
 };
 
-export const providerLogoMap: Record<string, string> = {
```

Comment:
please only change this section. nothing else is needed to change correct

Reply:
done, 
sorry.. my linter run during commit